Project.toml

```diff
@@ -25,3 +32,8 @@ Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [targets]
 test = ["CodecZlib", "Graphs", "EzXML", "ParserCombinator", "Test"]
+
+[weakdeps]
+CodecZlib = "944b1d66-785c-5afd-91f1-9de20f533193"
+EzXML = "8f5d6c58-4d21-5cfd-889c-e3ad7ee6a615"
```

Comment:
Shouldn't we also add `[compat]` entries for these packages?

Reply:
Yes, generally whatever goes to `[weakdeps]` must also be in `[compat]`. However I realize `CodecZlib` was not in compat even before. Are you aware of what would be the appropriate compat there ? If not we could put the current version which is `v0.7`.

Reply:
I think previously the extras were just there for the tests, and there a compat version was not required by JuliaRegistrator. I think putting `v0.7` or `v0.7.1` is fine, `CodecZlib` seems to be quite backwards compatible.

Reply:
I put `v0.7`